SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -206,9 +206,25 @@ bool FLocalDeploymentManager::LocalDeploymentPreRunChecks()
 	return bSuccess;
 }
 
-void FLocalDeploymentManager::TryStartLocalDeployment(FString LaunchConfig, FString RuntimeVersion, FString LaunchArgs,
-													  FString SnapshotName, FString RuntimeIPToExpose,
+void FLocalDeploymentManager::TryStartLocalDeployment(const FString& LaunchConfig, const FString& RuntimeVersion, const FString& LaunchArgs,
+													  const FString& SnapshotName, const FString& RuntimeIPToExpose,
 													  const LocalDeploymentCallback& CallBack)
+{
+	int NumRetries = 3;
```

Comment:
Probably want this to be a configurable setting? Can then change it in the test framework a bit easier

Reply:
Would love to... wanted to put it in editor settings... alas... special services.

Will add it to the header so it is a bit more easy to configure.